pages/common/prettier.md

```diff
@@ -0,0 +1,24 @@
+# prettier
+
+> An opinionated code formatter for JavaScript, JSON, CSS, YAML, and more.
+> More information: <https://prettier.io/>.
+
+- Format a file and print the result to stdout:
+
+`prettier {{path/to/file}}`
+
+- Check if a specific file has been formatted:
+
+`prettier --check {{path/to/file}}`
+
+- Run with a specific configuration file:
+
+`prettier --config {{path/to/config_file}} {{path/to/file}}`
+
+- Format a file in place:
```

Comment:
I don't like how this is worded, and I can't tell what it does so I can't suggest anything

Reply:
I don't it at all. @sbrl, could you help with this?

Reply:
Hrm, let me see. It seems fine to me, but if others don't like it what about this?
```suggestion
- Format a file, replacing the original file with the formatted code:
```

Reply:
`In place` could be a bit problematic, because I don't think many know that phrase. 
Another shorter suggestion:
```suggestion
- Format a file, replacing the original one:
```

Reply:
what about

```suggestion
- Format a file, replacing the original: